docs/pages/build/zero-to-one/build-your-first-mru.mdx

```diff
@@ -75,54 +78,32 @@ DATABASE_URI=<DATABASE_URI>
 ```
 
 :::info
-`PRIVATE_KEY` is the private key of account you want to register and deploy the rollup with.
-
-We don't store your private key anywhere. It's only used to sign the transactions and is stored in your local environment.
-:::
-
-- Now let's dive into the `stackr.config.ts` file.
-
-  :::warning
-  Ignore the warning for `./deployment.json` file not found. We will create this file in the next step.
-  :::
-
-- The `domain` in the config is used to create EIP712 Typed Signature over the inputs of actions. Feel free to tune this as per your requirements.
-
-```ts [stackr.config.ts]
-domain: {
-  name: "<APP_NAME>", // [!code focus]
-  version: "<APP_VERSION>", // [!code focus]
-  salt: "0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef" // [!code focus]
-}
-```
-
-## Registering your Micro-rollup
-
-Now that we have set up the config, let's register the Micro-rollup.
-Make sure you are in the project directory and run the following command:
-
-:::code-group
+`PRIVATE_KEY` is the private key of account you want to register and deploy the rollup with. This address acts as the Operator.
 
-```bash [npm]
-npx @stackr/cli@latest register
-```
-
-```bash [bun]
-bunx @stackr/cli@latest register
-```
-
-:::
-
-:::info
-This command will register your application on the registry contract. And update the `deployment.json` file with the latest values of `appId`, `appInbox` and `chainId`.
+We don't store your private key anywhere, it is stored in your local environment. It's only used to sign on incoming actions and proposed blocks.
 :::
 
-You'll see a message that the application has been registered on the registry.
+### Setup `stackr.config.ts`
 
-```bash [Terminal]
-✔ 📝 Registered application on registry
-✔ 🧰 Updated deployment.json
-```
+- Now let's dive into the `stackr.config.ts` file.
+- The main properties you might wish to tune here are `blockSize` and `blockTime` of your micro-rollup.
+You can also set `allowEmptyBlocks` to `true` if you want your micro-rollup to always produce blocks (even when there are no actions).
```

Comment:
```suggestion
You can also set `allowEmptyBlocks` to `true` if you want your micro-rollup to always produce blocks (even when there are no actions). This will produce blocks every `blockTime` milliseconds.
```